interface/main/calendar/find_appt_popup.php

```diff
@@ -30,6 +30,7 @@
  include_once("$srcdir/patient.inc");
  ///////
  require_once(dirname(__FILE__)."/../../../library/appointments.inc.php");
+require_once($GLOBALS['incdir']."/main/holidays/Holidays_Controller.php");
```

Comment:
Recommend only using lowercase in script file names (unless using classes that require this).